README.md

```diff
@@ -152,6 +152,7 @@ const chromeless = new Chromeless({
 - [`mousedown(selector: string)`](docs/api.md#api-mousedown)
 - [`mouseup(selector: string)`](docs/api.md#api-mouseup)
 - [`scrollTo(x: number, y: number)`](docs/api.md#api-scrollto)
+- [`scrollToElement(selector: string)`](docs/api.md#api-scrollto)
```

Comment:
Looks like this is linking to the wrong place.  Should be `docs/api.md#api-scrolltoelement`.